extensions/src/main/java/org/apache/falcon/extensions/mirroring/hdfs/HdfsMirroringExtension.java

```diff
@@ -51,6 +54,11 @@ public void validate(final Properties extensionProperties) throws FalconExceptio
     }
 
     @Override
+    public String getDescription() {
```

Comment:
yes will do.This was we will hardcode it that reading through the json only in case of trusted extensions, which should be fine.